.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -9,7 +9,6 @@ about: Something is broken? 🔨
 
 |    Q        |   A
 |------------ | ------
-| BC Break    | yes/no
 | Version     | x.y.z
```

Comment:
A table with one entry feels weird now.

Reply:
A form would be better in this case. A table provides the options but doesn't request a choice.